pkg/config/model/viper.go

```diff
@@ -806,3 +892,9 @@ func (c *safeConfig) GetProxies() *Proxy {
 	c.proxies = p
 	return c.proxies
 }
+
+func (c *safeConfig) ExtraConfigFilesUsed() []string {
+	c.Lock()
+	defer c.Unlock()
+	return c.loadedExtraConfigFilePaths
```

Comment:
I think we should return a copy to prevent the caller from being able to modify the internal slice.

Reply:
Good point ! c29636632ae9810f7ff065b4514e0ad31754ad7f.